src/PowerShellEditorServices/Services/Workspace/Handlers/ConfigurationHandler.cs

```diff
@@ -146,5 +123,7 @@ public void SetCapability(DidChangeConfigurationCapability capability)
         {
             _capability = capability;
         }
+
+        public event EventHandler<LanguageServerSettings> ConfigurationUpdated;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused event 'ConfigurationUpdated' or invoke it.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4961943)